### PR TITLE
whitelist pam_systemd_home.so for systemd-experimental

### PR DESCRIPTION
**Pull request by jsegitz**
audited in 1185285

### CODE REVIEW

File: configs/openSUSE/pam-modules.toml [hunk 1]
```diff
@@ -210,6 +210,14 @@ nodigests = [
     "*/security/pam_systemd.so",
 ]
 
+[[FileDigestGroup]]
+package = "systemd"
+type = "pam"
+note = "Module in systemd-experimental. This was not audited, if it's moved to a different binary rpm it needs to be audited first"
```
Comment (wfrisch) [inline configs/openSUSE/pam-modules.toml:216]: It appears to have been audited in the meantime:
https://bugzilla.opensuse.org/show_bug.cgi?id=1185285

**Comment by thesp0nge** [comment]
Hi @jsegitz, this PAM module was audited some months ago: https://bugzilla.opensuse.org/show_bug.cgi?id=1185285

File: configs/openSUSE/pam-modules.toml (hunk 1 above)
Comment (wfrisch) [reply configs/openSUSE/pam-modules.toml:216]: We can change it to
`note = "Module in systemd-experimental."`

Comment (thesp0nge) [reply configs/openSUSE/pam-modules.toml]: I changed the "note" field and added the bugzilla ticket number!

**Review by wfrisch — APPROVED** [review]

**Pull request merged**